src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,23 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: { title, user, userId, body, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      {user && <UserInfo user={user} key={userId} />}
```

Comment:
you don't need attribut `key` here, remove it
```suggestion
      {user && <UserInfo user={user}  />}
```